hbase-client/src/main/java/org/apache/hadoop/hbase/HRegionInfo.java

```diff
@@ -752,9 +785,17 @@ public static HRegionInfo convert(final HBaseProtos.RegionInfo proto) {
     // RegionInfo into HRegionInfo which is what is wanted here.
     HRegionInfo hri;
     if (ri.isMetaRegion()) {
-      hri = ri.getReplicaId() == RegionInfo.DEFAULT_REPLICA_ID ?
+      //TODO francis room for streamlining this logic
+      hri = ri.getReplicaId() == RegionInfo.DEFAULT_REPLICA_ID && ri.getRegionId() == 1
```

Comment:
What's the thinking here? Replicas for ROOT too?

Reply:
Yeah since we had it for meta. Which should have it for catalog as a whole too.